runtime/dart_isolate.cc

```diff
@@ -633,10 +633,16 @@ bool DartIsolate::UpdateThreadPoolNames() const {
   }
 
   if (auto task_runner = task_runners.GetPlatformTaskRunner()) {
+    bool is_merged_platform_ui_thread =
```

Comment:
Maybe we should have a static method that returns if current thread is the platform main thread (with platform specific implementations for checks).

Reply:
what sort of platform specific checks would we perform?